.travis.yml

```diff
@@ -50,3 +50,15 @@ deploy:
     access_key_id: AKIAIG6T67NMROEIMC7A
     secret_access_key:
       secure: WygEvdxSQNZpJnPtmk8GDK6ueHUO0tissYQU3rV3ZCQLNjPwPSYxuqEwno2zP07XPQBxzD2TTZRKQYY5q3st14wdkoaKyiEo97VF5kitQCHK6aCs2L9MUPfL6SjhE2KFzJVDdSJzAYQcxoSPps5mqqcvceYYPhgVg3sUriK7c10=
+  - provider: elasticbeanstalk
+    skip_cleanup: true
+    bucket_name: org-sagebridge-deployment-uat
+    zip_file: target/universal/bridgepf-0.1-SNAPSHOT.zip
+    region: us-east-1
+    app: BridgePF
+    env: BridgePF-uat
+    on:
+      branch: uat
+    access_key_id: AKIAIG6T67NMROEIMC7A
+    secret_access_key:
+      secure: WygEvdxSQNZpJnPtmk8GDK6ueHUO0tissYQU3rV3ZCQLNjPwPSYxuqEwno2zP07XPQBxzD2TTZRKQYY5q3st14wdkoaKyiEo97VF5kitQCHK6aCs2L9MUPfL6SjhE2KFzJVDdSJzAYQcxoSPps5mqqcvceYYPhgVg3sUriK7c10=
```

Comment:
You actually don't need a separate deployment provider. You can just use the same provider and just have different setings per branch. See for example https://github.com/Sage-Bionetworks/Bridge-Exporter/blob/develop/.travis.yml This way, if we need to change the Elastic Beanstalk deployment, we can change it in one place instead of in 3.